grawlix-cms/_admin/book.page-create.php

```diff
@@ -287,10 +287,76 @@
 
 
 
+// Script for adding some HTML tags into the post
 $blog_output = <<<EOL
 <label for="blog_headline">Headline</label>
 <input type="text" name="blog_headline" id="blog_headline" value="$blog_headline"/>
 <label for="blog_post">Post</label>
+
+<script language="javascript" type="text/javascript">
```

Comment:
Might be nicer to move the script to a separate `.js` file, mark the `textarea`s with class and have something like the following https://jsfiddle.net/jtojnar/u5o8ajzh/10/